app/javascript/scss/editor/_translation-settings.scss

```
@@ -13,11 +17,24 @@
   background: $bg-darker;
 }
 
+.translation-settings__keys-container {
+  height: 100%;
+  overflow-y: auto;
+  margin-inline: -1rem;
+  padding-inline: 1rem;
+
```

I'd prefer this use regular margins and padding, it's not like we care about different writing modes.

```suggestion
  margin: 0 -1rem;
  padding: 0 1rem;

```

---

Those are not truly equal though: `margin/padding-inline` only changes the margin/padding on the (in english) horizontal plane, while yours also sets them to zero on the vertical plane.

I didn't care about writing modes either, I was just using the shorthand that would avoid repeating the margin and padding for both left and right, while also avoiding overwriting the margin/padding in other axis.

But I'm fine with your suggestion regardless.
I honestly thought you'd point out the negative margin 😆. But I take you know why I did that? Maybe I should add a comment.

---

Yep, that's totally fine, to counteract the wrapper padding

---

To clarify: you still prefer `margin/padding: 0 n` over `margin-inline: n`?